pom.xml

```diff
@@ -1384,6 +1384,12 @@
                 <version>${project.version}</version>
             </dependency>
 
+            <dependency>
```

Comment:
We can't add a governance dependency to the framework. You can use a service stub to invoke recovery service instead.